.deps

```diff
@@ -0,0 +1,586 @@
+errors
```

Comment:
How was this file generated? When would it have to be updated? Maybe add a comment somewhere explaining that.

Is it Docker specific? If so, perhaps the file name could be adjusted to reflect that it's for the benefit of Docker, since otherwise it could appear as if it's part of some official golang dependency management solution.

Reply:
Added comment inside docker file:
```
# .dockerbuild.deps contains list of packages used by go-client
# needed to build them before building our code so it will be cached
# by docker and reused on next builds.
# it was generated manually by running `go build -v` and capturing output
# should be updated only if we remove some dependency from vendor dir
```
It was best way I was able to find to make docker to cache out build. I can't just precompile `vendor` dir because it contains many things that just do not compile.

Reply:
> I can't just precompile `vendor` dir because it contains many things that just do not compile.

Should we remove those? The vendor dir really needs some cleanup anyway. Preferably we'd get rid of the whole thing with golang modules...

Reply:
The way to precompile whole dir is like this ``
But this leads to errors like these:
```
# github.com/livepeer/go-livepeer/vendor/github.com/ethereum/go-ethereum/crypto/bn256
vendor/github.com/ethereum/go-ethereum/crypto/bn256/bn256_fast.go:26:6: G1 redeclared in this block
	previous declaration at vendor/github.com/ethereum/go-ethereum/crypto/bn256/bn256.go:30:6
```
Probably it's because it's tries to compile whole package, and I don't know how to get over it without manually removing files one-by-one from inside package - and this is obviously not good idea.